File: U4/C#/U4P_4.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U4P_4
    {
        static void Main(string[] args)
        {
            int[] numeros = { 5, -2, 0, -8, 10, -4, 7, -1 };

            // Contador para almacenar el total de números negativos
            int totalNegativos = 0;

            // Recorrer el arreglo y contar los números negativos
            foreach (int numero in numeros)
            {
                if (numero < 0)
                {
                    totalNegativos++;
                }
            }

            // Imprimir el resultado
            Console.WriteLine("El total de números negativos en el arreglo es: " + totalNegativos);
        }
    }
}

File: U4/Java/U4P_4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompan.u4p_4;

/**
 *
 * @author alfre
 */
public class U4P_4 {

    public static void main(String[] args) {
        int[] numeros = { 5, -2, 0, -8, 10, -4, 7, -1 };

        // Contador para almacenar el total de números negativos
        int totalNegativos = 0;
        for (int numero : numeros) {
            if (numero < 0) {
                totalNegativos++;
            }
        }
        System.out.println("El total de números negativos en el arreglo es: " + totalNegativos);
    }
}
